app/src/components/landing/pricing/PricingIndexPage/PricingPageFooter/index.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from "react";
+import { Row, Col, Typography, Space } from "antd";
+import { EVENTS, trackRequestDocumentClicked } from "./analytics";
+import RequestDocsModal from "./RequestDocsModal";
+import { getFunctions, httpsCallable } from "firebase/functions";
+
+const { Title, Text, Link } = Typography;
+
+const styles = {
+  footer: {
+    padding: "40px 20px",
+  },
+  container: {
+    maxWidth: "70vw",
+    margin: "0 auto",
+  },
+  title: {},
+  companyInfo: {
+    marginTop: "20px",
+    textAlign: "center",
+    color: "var(--neutrals-gray-300)",
+  },
+};
+
+const PricingPageFooter = () => {
+  const [isRequestDocsModalOpen, setisRequestDocsModalOpen] = useState(false);
+
+  const handleDocRequiredOnClick = () => {
+    trackRequestDocumentClicked();
+    const salesInboundNotification = httpsCallable(getFunctions(), "salesInboundNotification");
+    try {
+      salesInboundNotification({
+        notificationText: EVENTS.REQUEST_DOCUEMNT_CLICKED,
+      });
+    } catch (error) {
+      console.error(error);
+    }
+    setisRequestDocsModalOpen(true);
+  };
+  return (
+    <>
+      <div style={styles.footer}>
+        <div style={styles.container}>
+          <Row justify="space-between" align="top" gutter={[16, 16]}>
+            <Col xs={24} md={6}>
+              <Title level={4} style={styles.title}>
+                Get Requestly
+              </Title>
+              <Space direction="vertical">
+                <Link target="_blank" href="https://requestly.com/desktop/">
+                  Desktop app
+                </Link>
+                <Link target="_blank" href="https://requestly.com/downloads/chrome/">
+                  Chrome Extension
+                </Link>
+                <Link target="_blank" href="https://requestly.com/downloads/edge/">
+                  Edge Addon
+                </Link>
+                <Link target="_blank" href="https://requestly.com/integrations/requestly-for-selenium/">
+                  Selenium Addon
+                </Link>
+                <Link target="_blank" href="https://requestly.com/downloads/for-safari/">
+                  For Safari
+                </Link>
+              </Space>
+            </Col>
+            <Col xs={24} md={6}>
+              <Title level={4} style={styles.title}>
+                Resources
+              </Title>
+              <Space direction="vertical">
+                <Link target="_blank" href="https://requestly.com/privacy/">
+                  Privacy Policy
+                </Link>
+                <Link target="_blank" href="https://requestly.com/terms/">
+                  Terms of Service
+                </Link>
+                <Link target="_blank" href="https://requestly.com/blog/">
+                  Blog
+                </Link>
+                <Link target="_blank" href="https://developers.requestly.com/">
```

Comment:
there is an existing constant for this url in the LINKS constant object called REQUESTLY_DOCS